### PR TITLE
Update Django, Ray, Ubuntu

### PR DESCRIPTION
**Pull request by jhazentia**
This includes the following. Thanks to Doug Boulware for Ray compatibility updates.

- Update contact info
- Update Django to 4.2 or newer (less than 5), ray to 2.10 or newer (tested using 2.37)
- Changes to support Ubuntu 22.04 (get_current_cpu_clock_speed() updated)

Tested performance of new ray version by measuring run time of CBRS measure action using perf_counter immediately before and after action in scheduler. Repeated the action 5 times. Average run time with old ray was 103.16 seconds, new version average run time was 102.438 seconds.

### CODE REVIEW

File: scos_actions/actions/acquire_sea_data_product.py [hunk 1]
```diff
@@ -585,35 +581,42 @@ def __call__(self, sensor: Sensor, schedule_entry: dict, task_id: int):
...
-                    channel_data.extend(data)
+            # Now block until the data is ready
+            dp_refs_tuple = ray.get(data_products_refs[index])
+            psd_ref, pvt_ref, pfp_ref, apd_ref = dp_refs_tuple
+            psd_data = ray.get(psd_ref)
+            channel_data.extend(psd_data)
+
+            pvt_data = ray.get(pvt_ref)
+            # Power-vs-Time results, a tuple of arrays
+            data, summaries = pvt_data  # Split the tuple
```
Comment (aromanielloNTIA) [inline scos_actions/actions/acquire_sea_data_product.py:597]: I haven't tested to confirm, but it appears that the PVT data (`data` after the "split the tuple" line) is never added to the `channel_data` array and thus would be missing from the output data product.

**Review by aromanielloNTIA — COMMENTED** [review]
I will test this next week, but I think I noticed that the PVT data is not added to the sigmf-data file after these changes

File: scos_actions/actions/acquire_sea_data_product.py (hunk 1 above)
Comment (jhazentia) [reply scos_actions/actions/acquire_sea_data_product.py:599]: Good catch. Please review fix in 617d04f. With the fix, the data product analytics graphs appear to be correct when monitoring through manager. Also sigmf file sizes when compared to current tagged version appear to be correct.  Also looked at performance again which still looks good. Average run time with old ray was 107.304 seconds, new version average run time was 106.85 seconds.

**Review by aromanielloNTIA — APPROVED** [review]
Changes look good, tested and checked data output on seadog01. Let's hold off on creating a new release until #124  and #125  are merged as well.

**Pull request merged**